app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -77,8 +74,8 @@ const MakeDecision = ({ data }) => {
     [COPY_FROM_HISTORY_LINE]: false,
     [SELECT_DEFECT_MANUALLY]: true,
   });
-  const [step, setFormStep] = useState(CONFIGURATION);
   const [modalHasChanges, setModalHasChanges] = useState(false);
+  const [rightSectionIsLess, setIsShown] = useState(true);
```

Comment:
Can we encapsulate setIsShown state into modal? I suggest to pass setRightSectionCollapsed and rightSectionCollapsed as a parameters to the renderSideSection method and don't repeat the states into different components.

Reply:
This parameter must also be available in the main modal content (left section), so it must be at the top level.
Another way I see it is to render the `children` of the `modal layout` as a render prop, but this will complicate the modal content.

Reply:
How i can provide ightSectionCollapsed  to function renderTitle, renderRightSection, getAccordionTabs?

Reply:
I guess via arguments. Like:
`const renderRightSection = (rightSectionCollapsed) => {
...
}`